golang/Makefile

```diff
@@ -24,7 +24,7 @@ NATIVE_SRC = tvm_runtime_pack.cc
 
 GOPATH=$(CURDIR)/gopath
 GOPATHDIR=${GOPATH}/src/${TARGET}/
-CGO_CPPFLAGS="-I. -I${TVM_BASE}/ -I${TVM_BASE}/3rdparty/dmlc-core/include -I${TVM_BASE}/include -I${TVM_BASE}/3rdparty/dlpack/include/"
+CGO_CPPFLAGS="-I. -I${TVM_BASE}/ -I${TVM_BASE}/3rdparty/compiler-rt/ -I${TVM_BASE}/3rdparty/dmlc-core/include -I${TVM_BASE}/include -I${TVM_BASE}/3rdparty/dlpack/include/"
```

Comment:
Do not include the compiler RT part given that we can directly use relative include in the src and compiler rt is not necessary as part of the header file

Reply:
inside the cc file:

```
#include "../../3rdparty/compiler-rt/builtin_fp16.h"
```